modules/ROOT/pages/activity-tracking.adoc

```diff
@@ -25,30 +25,45 @@ Use the search box to find the transmission based on criteria such as:
 * Filename
 * Control number
 * Endpoint
-* Date the file was received
-+
-Use the filters to find transmissions based on:
-+
+* Date the file was received +
+
+. You can then use the filters to find transmissions based on:
+
 * *Type* +
 Type of message transmission:
 ** *Both directions*
 ** *Sent to partners*
 ** *Received from partners*
-* *From Date*
-Filter for transmissions that sent or received after this date. 
+* *From Date* +
+Filter for transmissions that sent or received after this date.
 * *To Date* +
-Filter for transmissions that are sent or received up to this date. 
+Filter for transmissions that are sent or received up to this date.
 * *Ack Status* +
 Filter for transmissions based on their acknowledgment status:
 ** *All*
 ** *Overdue*
-** *Rejected*
-. Select the transmission that matches your search criteria and review the summary information: 
+** *Rejected* +
+* *Message Processing Status* +
+Filter for transmissions based on their message processing status:
+** *All*
+** *Delivered*
+** *Errored*
+** *Processing*
+** *Received*
+* *Partner name* +
+Filter for transmissions based on your partner's name.
+* *Partner message type* +
+Filter for transmissions based on the message type, such as `X12` or `850`. +
+[NOTE]
+For X12 custom message types, partner names should not be considered.
```

Comment:
Please remove this note. The message type values shown in the drop down do not actually map to the partner message type names anyways.

Reply:
Removed!